src/components/Navbar.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from "react";
+
+import Brand from './Brand.jsx';
+import Logo from './Logo.jsx';
+import Navbutton from './Navbutton.jsx';
+
+class NavBar extends Component { 
+  cunstructor(props) {
```

Comment:
@Albosonic we are going to be using redux, with that in mind hopefully most of our comps are stateless (react state that is).